simulator/Vagrantfile

```diff
@@ -110,8 +110,9 @@ Vagrant.configure("2") do |config|
     config.vm.provision "file", source: "native_streaming.service", destination: "/tmp/native_streaming.service"
     config.vm.provision "shell", inline: "cp /tmp/*.service /etc/avahi/services/"
 
-    # Copy opendaq-config.json file 
-    config.vm.provision "file", source: "opendaq-config.json", destination: "/tmp/opendaq-config.json"
+    # Copy opendaq-config.json file
+    config.vm.provision "file", source: "opendaq-config.json", destination: "/tmp/"
```

Comment:
Why don't you copy directly to `/home/opendaq/opendaq`?

Reply:
@Unsttopabull Copying directly to the home directory requires appropriate access rights, which are not granted within the Vagrant provisioning file copying process. Therefore, all necessary files are first copied into the `/tmp` directory as an intermediary step.

Reply:
It that really the case?
I don't see a problem in the [documentation](https://developer.hashicorp.com/vagrant/docs/provisioning/file) as it also copies to the home dir.

Reply:
I guess the documentation example only covers copying files into `/home/vagrant`, which is the home directory of the user `vagrant` logged in via SSH as part of the provisioning process. However, in our scenario, we create a new user and need to copy files into its home directory.